THIRD-PARTY-LICENCES.txt

```diff
@@ -0,0 +1,1689 @@
+Google Ads API Ruby Library  uses third-party libraries or other resources that
```

Comment:
Since the distributed `.gem` will include all the source files, will it automatically include this file as well?